### DISCUSSION TITLE
How to enable cloud functions??

### DISCUSSION THREAD

**Discussion by catrawalkar** [AntlerVC/firetable#422] [Support / Q&A] [answered]
How do I enable cloud functions? 

How do I use Typeform ?

Is there any detailed documentation about how to Use?  I find many features which are not documented.

**Comment by harinij** [comment, marked as answer]
Closing this older issue. We have all the documentation available here now: [Rowy docs](http://docs.rowy.io/)

Specifically, for typeform we have a video tutorial available [here](https://www.youtube.com/watch?v=icH_YfG70ck)
